app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,37 @@
+ <div class="foods-container">
+  <div class="foods-header">
+    <h1 class="foods-title">Recipes</h1>
+    <div>
+        <%= link_to "New Recipe", new_user_recipe_path(current_user), class: 'add-food-btn' %>
+    </div>
+  </div>
+  <div class="foods-table">
+    <table>
+      <thead>
+        <tr>
+          <th class="foods-list-col-food">Recipes</th>
+        </tr>
+      </thead>
+      <% if @recipes.empty? %>
+        <tbody>
+          <tr class="foods-list-empty">
+            <td colspan="4" class="foods-list-empty-text">Please Add Recipes!!!</td>
+          </tr>
+        </tbody>
+      <% else %>
+        <tbody>
+          <% @recipes.each do |recipe| %>
+            <tr class="foods-list-row">
+              <td class="foods-list-col-food recipe">
+                <h2><%= link_to recipe.name, user_recipe_path(current_user, recipe.id) %></h2>
+                <%= form_for recipe, url: user_recipe_path(current_user, recipe), method: :delete do |f| %>
+                    <%= f.submit "Remove", data: { confirm: "Are you sure?" } %>
+                <% end %>
```

Comment:
- [ ] Excellent work your design is 👌 however you need to make sure you follow the wireframe layout(you can be creatif on the design but the layout should be the same)

Your layout | wireframe layout 
----|---
![image](https://user-images.githubusercontent.com/65068771/229274311-e9392773-0351-434f-98db-6e9cd1bca525.png) | ![image](https://user-images.githubusercontent.com/65068771/229274362-35cf8878-1fdf-4c00-a61a-12bae6851441.png)